docs/specifications/tests/DNSSEC-TP/dnssec13.md

```diff
@@ -107,36 +115,33 @@ been created by a DNSKEY from the zone's DNSKEY RRset.
             1. If the key ID from the RRSIG is not a member of the
                set *DNSKEY Key ID*, then output
                *[RRSIG_NOT_MATCH_DNSKEY]* and go to next RRSIG.
-            2. If the RRSIG cannot be verified by the DNSKEY from
+            2. If the Zonemaster installation does not have support for
+               the algorithm that created the RRSIG, then add the
+               DNSKEY key tag and algorithm to the
+               *Unsupported Algorithm* set.
+            3. If the RRSIG cannot be verified by the DNSKEY from
                *DNSKEY Record* with the matching RRSIG key ID, then
                output *[RRSIG_BROKEN]*.
-6. If *DNSKEY Algorithm* is non-empty for at least some name server IP
-   addresses in *NS IP* and none of the following messages were
-   outputted, then output *[ALL_ALGO_SIGNED]*:
-   * *[ALGO_NOT_SIGNED_RRSET]*
-   * *[NO_RESPONSE_RRSET]*
-   * *[RRSET_NOT_SIGNED]*
-   * *[RRSIG_BROKEN]*
-   * *[RRSIG_NOT_MATCH_DNSKEY]*
 
+7. If the *Unsupported Algorithm* set is non-empty, then for each
+   DNSKEY key tag output *[DS13_ALGO_NOT_SUPPORTED_BY_ZM]* and print
```

Comment:
I'm not sure what the perfect tag would be, but did you consider the verb implemented?

Reply:
Is "implemented" correct if we consider the situation where it depends of the version of OpenSSL?